### DISCUSSION TITLE
Is there any way to change the color of hostname and username?

### DISCUSSION THREAD

**Discussion by 7549brijesh** [kovidgoyal/kitty#5195] [Q&A] [answered]
I just want to customize the color of hostname and username its way to simple(typically white color)
![2022-06-13_21-46](https://user-images.githubusercontent.com/25589680/173398569-c6dfa39f-8d0e-46df-9676-d6b36005337e.png)

someone here to help??

**Comment by kovidgoyal** [comment, marked as answer]
This has to with the configuration of your shell not the terminal
emulator. And if you are using Debian (which for some stupid reason
known only to itself, screws up its shell rc files, follow #4590)

**Reply by 7549brijesh** [reply]
so for changing the color I have to do the setting in shell not on the terminal emulator?
Right?
I am an Arch user(endeavour os)

**Reply by kovidgoyal** [reply]
yes, right. google coloring my shell prompt

**Reply by 7549brijesh** [reply]
last question
Thus this will reflect to the kitty terminal as well?(if i do the changes in the shell)
Thanks for the help!!
